### PR TITLE
Arc 1558 push app id to queue msg  other events

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
Add `gitHubAppConfig` to branch and deployment events payload

**Why**
Current installationId is not unqiue.

**Added feature flags**
N/A

**Affected issues**  
ARC-1558

**How has this been tested?**  
Unit test.

**Whats Next?**
Backfill events.

### CODE REVIEW

File: src/github/branch.test.ts [hunk 1]
```diff
@@ -1,164 +1,78 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (gxueatlassian) [inline src/github/branch.test.ts:1]: Move test to a proper place.

File: src/github/branch.test.ts [hunk 2]
```diff
@@ -1,164 +1,78 @@
...
-import { createWebhookApp } from "test/utils/probot";
```
Comment (gxueatlassian) [inline src/github/branch.test.ts:2]: Purely move the file to a proper location, and use the same name for unit testing.

File: src/github/deployment.test.ts [hunk 3]
```diff
@@ -1,177 +1,78 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (gxueatlassian) [inline src/github/deployment.test.ts:1]: Purely move the file to a proper location, and use the same name for unit testing.

**Review by rachellerathbone — APPROVED** [review]

**Review by mboudreau — APPROVED** [review]
Nice.

**Pull request merged**